src/main/resources/org/jenkins/plugins/lockableresources/actions/LockableResourcesRootAction/index_af.properties

```
@@ -51,9 +54,11 @@ btn.reserve.detail=Reserves an available resource for currently logged user inde
 btn.unReserve=Unreserve
 btn.unReserve.detail=Un-reserves a resource that may be reserved by some person already.<br>The user can unreserve only own resource. Administrator can unreserve any resource.
 btn.reassign=Reassign
-btn.reassign.detail=Reserves a resource that may be or not be reserved by some person already.<br>Giving it away to currently logged user indefinitely (until that person, or some explicit scripted action, decides to release the resource).
+btn.reassign.detail=Reserves a resource that may be or not be reserved by some person already.<br>Giving it away to currently logged user indefinitely (until that person, or some explicit scripted action, decides to release the resource).<br>This action can be done only by administrator.
```

EX. this is wrong and shall be changed by next documentation update.
```suggestion
btn.reassign.detail=Reserves a resource that may be or not be reserved by some person already.<br>Giving it away to currently logged user indefinitely (until that person, or some explicit scripted action, decides to release the resource).
```

---

Pls do not commit the suggestion, I want just mark the failure here.